src/ui/MainMenu.cpp

```diff
@@ -10692,16 +12579,19 @@ namespace MainMenu {
             {
 #endif
                 // lobby list has returned
-                closeText("lobby_list_request");
+                closePrompt("lobby_list_request");
 
 #if defined(STEAMWORKS)
 	            for (Uint32 c = 0; c < numSteamLobbies; ++c) {
+	                auto lobby = (CSteamID*)lobbyIDs[c];
+	                auto pchFlags = SteamMatchmaking()->GetLobbyData(*lobby, "svFlags");
+	                auto flags = (int)strtol(pchFlags, nullptr, 10);
 	                LobbyInfo info;
 	                info.name = lobbyText[c];
 	                info.players = lobbyPlayers[c];
-	                info.ping = 100; // TODO
-	                info.locked = false; // TODO
-	                info.flags = 0; // TODO
+	                info.ping = 50; // TODO
+	                info.locked = false; // this will always be false because steam only reported joinable lobbies
```

Comment:
Need to add lobby attribute for game in session, EOS uses CURRENTLEVEL and then if currentlevel >= 0 report `LobbyHandler_t::EResult_LobbyFailures::LOBBY_GAME_IN_PROGRESS` Host needs to continually update this lobby attribute as per EOS, `EOSFuncs::LobbyData_t::updateLobbyDuringGameLoop()`

Need to add 'visibility' attribute as well, `PERMISSIONLEVEL` to show full lobbies

Reply:
Lobbies are closed when the game starts, otherwise yeah i would do that. We should probably go with the "every lobby is MAXPLAYERS + 1 in size" method